### PR TITLE
Add `react-hot-loader` and update text editors

### LINKED ISSUE #300: css issue for long text (rich text editor)
screen shot 2018-02-04 at 8 08 48 pm

### PR DESCRIPTION
**Pull request by aulneau**
This update does a few things:

- Adds [`react-hot-loader`](https://github.com/gaearon/react-hot-loader) to the dev project to allow for hmr.
- Updated the dev project to put the css in a separate file
- Along with moving the nav bar to the bottom and preventing it from causing the screen to scroll
- Commented out the ACCESS_TOKENS import to something that is not committed to the repo, (would cause errors immediately even for anyone not using Satellite Maps)
- ran [`fixpack`](https://github.com/HenrikJoreteg/fixpack) on `package.json` ✨
 - Fixed text editors stretching due to long lines/content -- resolves #300 

text-editor-fix

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "presets": ["react", "es2015"],
   "plugins": [
+    "react-hot-loader/babel",
     "transform-object-rest-spread",
```
Comment (VeraZab) [inline .babelrc]: thank you!!

File: dev/App.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+import plotly from 'plotly.js/dist/plotly';
+import createPlotComponent from 'react-plotly.js/factory';
+import PlotlyEditor from '../src';
+import '../src/styles/main.scss';
+import Nav from './Nav';
+
+// Satellite Maps in the Editor ->
+// https://github.com/plotly/react-plotly.js-editor#mapbox-access-tokens
+// import ACCESS_TOKENS from '../accessTokens';
+
```
Comment (VeraZab) [inline dev/App.js]: I'll send you the file and we can uncomment

File: dev/Nav.js [hunk 3]
```diff
@@ -30,6 +15,7 @@ const Nav = props => (
           label: item.name,
           value: i,
         }))}
+        className="open-top"
         searchable={true}
```
Comment (VeraZab) [inline dev/Nav.js]: thanks!!

**Comment by nicolaskruchten** [comment]
I'm late to the party here but I'll just mention that there's very little to gain in refactoring the `dev` app by splitting it up into multiple files etc... It's meant to be as easy to hack on as possible, and not to grow unduly :)

**Comment by aulneau** [comment]
Totally agree :) I tried to not add much, I mostly did it to wrap the app in the hot loader, and to locate all css modification to one place so people wouldn't have to look around. If it's too much though I'd be happy to remerge them 👍

**Comment by nicolaskruchten** [comment]
Definitely not worth spending additional time undoing it, just wanted to make sure we don't spend additional time cleaning that app up more :)

**Comment by nicolaskruchten** [comment]
But *wow* NICELY DONE on getting HMR to work here! I spent a good 30min on it and didn't manage to make it work... 👍 👍 👍

**Comment by nicolaskruchten** [comment]
I actually thought HMR was not possible with the way we are wrapping components etc... I'll have to backport this config to `react-pivottable`

**Comment by aulneau** [comment]
Yeah `react-hot-loader` is great -- make sure to pull from [`@next`](https://github.com/gaearon/react-hot-loader/tree/next) when adding it!

**Pull request merged**